evelyn_chen/alpha_challenges/conditionals.md

```diff
@@ -27,13 +43,34 @@ Create a Ruby method called `temperature_status` that takes three arguments: the
   - If the air conditioner is non-functional and the current temperature is below the the desired temperature, print the string "Fix the A/C whenever you have the chance. It's cool."
 
 ```ruby
-# write your code here
+def temperature_status(current_temp, AC_functional, desired_temp)
+  if AC_functional and current_temp >= desired_temp
```

Comment:
What if the AC is functional and the current temp is less than the desired temp?

Reply:
Also, do we want to check if it's greater than or equal to (>=)or just greater than (>)?

Reply:
I'll add that conditional as well, thanks for the feedback!

Reply:
No need for a new conditional, but you should be checking for the AC_fucntional in the other cases as well :)

Reply:
Ok, got it!